Tribler/Main/tribler_main.py

```diff
@@ -24,7 +24,7 @@
 import urllib
 
 original_open_https = urllib.URLopener.open_https
-import M2Crypto  # Not a useless import! See above.
+
```

Comment:
The `M2Crypto` import is already removed in `devel` so I'm not sure whether we should remove it here.